### PR TITLE
Add SUPPORT.md file to repo

### PR DESCRIPTION
**Pull request by jperrin**
Signed-off-by: Jim Perrin <Jim.Perrin@microsoft.com>

###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

Adds a SUPPORT.md file to the repository as part of project documentation

###### Change Log  

- Added SUPPORT.md file to repository for project documentation

###### Does this affect the toolchain?  

NO

### CODE REVIEW

**Review by PawelWMS — APPROVED** [review]

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+This project uses [GitHub issues][gh-issue] to [track bugs][gh-bug] and [feature requests][gh-feature]. Please search the existing issues before filing new issues to avoid duplicates. For new topics, file your bug or feature request as a new issue.
+
+For questions about using this project, please look at the [Demo repo for CBL-Mariner][demo] and our [Contributor's Guide][contributor] if you want to work on CBL-Mariner.
+
+## Microsoft Support Policy
+
+Support for CBL-Mariner is limited to the resources listed above.
+
+[gh-issue]: https://github.com/microsoft/CBL-Mariner/issues/new/choose
+[gh-bug]: https://github.com/microsoft/CBL-Mariner/issues/new?assignees=&labels=Issue-Bug&template=bug_report.md&title=
```
Comment (thcrain-msft) [inline SUPPORT.md:14]: It looks to me like the `gh-feature` and `gh-bug` links reference a couple of bug templates that I can't actually find in our `.github` folder. Additionally, when I go to those URLs, it doesn't fill out the labels like the query strings would imply.

**Comment by jperrin** [comment]
good catch. This doc was largely cut/paste from another inside msft, but I didn't pull in the template files. We can either change the wording in the doc, or I can see about bringing in those templates. Since that's more of a workflow shift I'll defer to the dev opinions on the right way to move forward.

File: SUPPORT.md (hunk 1 above)
Comment (thcrain-msft) [inline SUPPORT.md:14]:
```suggestion
[gh-bug]: https://github.com/microsoft/CBL-Mariner/issues/new?labels=bug
```

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+[gh-feature]: https://github.com/microsoft/CBL-Mariner/issues/new?assignees=&labels=Issue-Feature&template=Feature_Request.md&title=
```
Comment (thcrain-msft) [inline SUPPORT.md:15]:
```suggestion
[gh-feature]: https://github.com/microsoft/CBL-Mariner/issues/new?labels=enhancement
```

**Comment by thcrain-msft** [comment]
Sure thing- I don't mind the document's wording as is. We can add templates in later PRs as necessary. Since we have labels for bugs and feature requests already, I went ahead and made a couple of suggestion comments that will make the URL autofill those tags.

**Review by thcrain-msft — APPROVED** [review]

**Pull request merged**